docker-compose.yml

```diff
@@ -24,6 +25,7 @@ services:
       # comment out 2 following lines for team edition
       # args:
       #   - edition=team
+    #read_only: true
```

Comment:
I guess you can remove this commented line

Reply:
Indeed, I tryied to make it woks with read only but not mounted (ie: plugin folder) volumes were created with random UID/GID in the container.